packages/monitor-v2/src/price-publisher/PublishPrices.ts

```diff
@@ -0,0 +1,154 @@
+import { paginatedEventQuery } from "@uma/common";
+import { OracleHubEthers, OracleRootTunnelEthers, VotingV2Ethers } from "@uma/contracts-node";
+import {
+  ArbitrumParentMessenger,
+  OptimismParentMessenger,
+} from "@uma/contracts-node/dist/packages/contracts-node/typechain/core/ethers";
```

Comment:
is it a requirement to drill down all the way into the package like this or is it possible to include this import from `@uma/contracts-node` directly? if its not possible we should consider changing how we create the export from `contracts-node` such that you don't need to do this. (not something to do now or in this PR).